src/App.css

```diff
@@ -11,8 +11,9 @@ body {
 
 .Sidebar {
   height: 100%;
-  width: 18%;
+  width: 100%;
```

Comment:
we use 100% width, because now the width is controlled by the parent column
provided by reactstrap